src/generation/viterbi/viterbi_index.py

```diff
@@ -1,12 +1,14 @@
 from dataclasses import dataclass
+from typing import ClassVar
 
 from common.structures.chord import Chord, ChordQuality
 from common.structures.pitch import Pitch
 
 
 @dataclass
 class ViterbiIndex:
-    index = -1
+    TOTAL_STATES: ClassVar[int] = 12 * len(ChordQuality)
+    index: int = -1
 
     def __init__(self, index: int):
```

Comment:
We shouldn't need an init function -- @dataclass provides a default one to populate class fields.